### DISCUSSION TITLE
probleme instalation base dev

### DISCUSSION THREAD

**Discussion by ZykaOfficiel** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#316] [Q&A] [answered]
Bonjour je vien de tenter d'installer le serveur dev mais j'ai cette erreur qui apparait, auriez vous une solution ?
![image_2024-04-29_134226867](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/83522631/0cca6f9c-9479-42dc-9e0d-0b7581a6a166)

et je bloque a se niveaux sur fivem
![image_2024-04-29_134344094](https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/assets/83522631/5eb83d2f-ea4d-471a-9485-5ad458f9f82e)

**Comment by DreamXZE** [comment]
Hello 👋 
Tu n'as surement pas suivi les étapes indiqué ici:  https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server?tab=readme-ov-file#prepare-and-configure-the-server

On peut lire dans le message que soz_api.accounts n'existent pas sur ta DB 👍

**Comment by Laikker** [comment, marked as answer]
C'est normale de ne pas avoir soz_api.accounts car ca fait parti du system de white list qui n'est pas OpenSource.
Le vrai problème semble être que le soz-core ne demarre pas, surement car il n' a pas été buildé.
Donc je rejoins Dream, je pense que les étapes du build du soz-core n'ont pas été faite
